frontend/src/actions/parcelsActions.tsx

```diff
@@ -22,48 +22,74 @@ export const storeParcelsAction = (parcelList: IProperty[]): IStoreParcelsAction
 //Parcel Detail API action
 export interface IAddress {
   line1: string;
-  line2: string;
-  city: string;
-  province: string;
+  line2?: string;
+  city?: string;
+  cityId: number | any;
+  province?: string;
+  provinceId: string;
   postal: string;
 }
 
 export interface IBuilding {
   id: number;
+  localId: string;
   description: string;
   address: IAddress;
-  latitude: number;
-  longitude: number;
-  localId: string;
-  constructionType: string;
-  postal: string;
-  buildingFloorCount: number;
-  buildingPredominateUse: string;
+  latitude: number | any;
```

Comment:
I see a few of these `number | any` typings. Why use any here? Are we trying to express a nullable number here? If not, why not just using `any`?

Reply:
Yeah, this is functionally equivalent to any, but I have a slight preference for this because it shows the intended type of the field even if I'm required to make this an any to get the desired behaviour out of formik.